.github/workflows/artifacts_CI.yml

```diff
@@ -0,0 +1,65 @@
+name: Artifacts CI
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    tags: ["*"]
+    paths:
+      - "ray_julia_jll/Artifacts.toml"
+      - ".github/workflows/artifacts_CI.yml"
+  pull_request:
+    paths:
+      - "ray_julia_jll/Artifacts.toml"
+      - ".github/workflows/artifacts_CI.yml"
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on:
+      labels: ubuntu-20.04-16core
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - "1.8"  # Version used with MVP
+          - "1.9"  # Latest release
+        os:
+          - ubuntu-latest
+        arch:
+          - x64
+    env:
+      ray_julia_jll_dir: ./ray_julia_jll
+    steps:
+        - uses: actions/checkout@v3
+        - uses: julia-actions/setup-julia@v1
+          with:
+            version: ${{ matrix.version }}
+            arch: ${{ matrix.arch }}
+        - uses: julia-actions/cache@v1
+        - name: Install Ray CLI
+          run: |
+            pip install --upgrade pip wheel
+            PYTHON=$(python3 --version | perl -ne '/(\d+)\.(\d+)/; print "cp$1$2-cp$1$2"')
+            pip install -U "ray[default] @ https://github.com/beacon-biosignals/ray/releases/latest/download/ray-2.5.1-${PYTHON}-manylinux2014_x86_64.whl" "pydantic<2"
```

Comment:
I was thinking about the latest release and this will eventually break when we update to say `ray-2.7.0`. I don't have a good solution for this besides just using a static release.